docs/src/quick_start.rst

```diff
@@ -1,6 +1,342 @@
 .. _gv-quick-start:
 
-🚧 Quick Start
-==============
+.. include:: common.txt
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed accumsan maximus maximus. Vestibulum nec molestie lacus. Praesent.
+:fa:`truck-fast` Quick Start
+============================
+
+Now that you've :ref:`installed <gv-installation>` ``geovista``, let's take a
+quick tour to see some of the features and capabilities on offer.
+
+Resources
+---------
+
+For convenience, ``geovista`` comes with numerous **pre-canned resources** to
+help get you started on your visualization journey e.g., rasters, `VTK`_
+meshes, `Natural Earth Features`_, and various Earth Science sample data.
+
+``geovista`` will automatically download **versioned** resources on-the-fly,
+as and when she needs them. However, if you want to download and cache all
+``geovista`` resources to make them available offline, simply perform the
+following on the command line:
+
+.. code:: console
+
+    geovista download --all
+
+To view the manifest of registered resources:
+
+.. code:: console
+
+    geovsita download --list
+
+.. note::
+
+    Want to know more?
+
+    .. code:: console
+
+        geovista download --help
+
+
+Examples
+--------
+
+.. pyvista-plot::
+   :context:
+   :include-source: false
+   :force_static:
+
+   import pyvista
+   pyvista.set_jupyter_backend('static')
+   pyvista.global_theme.background = 'white'
+   pyvista.global_theme.window_size = [400, 300]
+   pyvista.global_theme.axes.show = False
+   pyvista.global_theme.anti_aliasing = 'fxaa'
+
+.. note::
+  :class: margin
+
+  The **Interactive Scene**
+  `vtk.js <https://kitware.github.io/vtk-js/index.html>`_ backend does **not**
+  support rendering text or points as spheres.
+
+Let's explore various atmospheric and oceanographic model data using
+``geovista``, which makes it easy to visualize **rectilinear**,
+**curvilinear**, and **unstructured** meshes.
+
+
+OISST AVHRR
+^^^^^^^^^^^
+
+This example renders a `NOAA/NCEI Optimum Interpolation SST`_ (OISST) Advanced
+Very High Resolution Radiometer (AVHRR)
+:term:`rectilinear grid <Rectilinear Grid>` containing ``1,036,800``
+quadrilateral :term:`cells <Cell>` and ``1,038,961`` :term:`points <Point>`.
+The grid is created from the bounds of ``1-D`` geographic longitude and
+latitude coordinates using the :meth:`~geovista.bridge.Transform.from_1d`
+method.
+
+A `threshold`_ is applied to the grid to remove ``345,650`` ``NaN``
+:term:`land mask <Land Mask>` cells, revealing a `NASA Blue Marble`_
+:term:`textured mapped <Texture Map>` :term:`base layer <Base Layer>`
+underneath.
+
+*Sea Surface Temperature* data located on the grid cells are rendered using
+the :term:`perceptually uniform <Perceptually Uniform Colormap>`
+`cmocean balance`_ diverging colormap, along with
+`10m Natural Earth coastlines`_.
+
+The *grid*, *coastlines* and *base layer* are then transformed to the
```

Comment:
There is no grid, if that means a graticule ??

Reply:
There is no graticule in this example, and I can see the confusion, so let's avoid that. I'll simply use the term `mesh`.

I was hanging on to the terminology of rectilinear and curvilinear being grids, and unstructured mesh ... although through `pyvista` we create a mesh in all cases.